RFC-by-stage/1-approved/distribute-css-as-unprefixed.md

```diff
@@ -0,0 +1,30 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Distribute CSS unminified and unprefixed
+
+## The issue to be solved
+
+CSS is currently being distributed as minified and vendor-prefixed assets.
+
+This is unnecessary since users (developing apps using npm) process styles for minification and vendor-prefixing, therefore css "as modules" should ship without being processed. This issue was raised by @liamfiddler [here](https://github.com/designsystemau/RFCs/pull/11#discussion_r724769922).
```

Comment:
Typically files from within the node_modules folders are not processed. That's the reason we publish transpiled babledfied js in our dists for react components. It's backwards and I agree it's not great but that's the norm these days. And one of the issues we have encountered in GOLD was that consumers were unable to use pancake because it included CSS files in their dist folders that were located within node_modules which most users were unable to write CSS loaders for their bundlers for one reason or another.